plugins/terraform/src/components/DenseTable/DenseTable.tsx

```diff
@@ -45,39 +45,77 @@ export const DenseTable = ({ data, isLoading, title }: DenseTableProps) => {
 
   const closeDialog = () => setIsOpen(false);
 
-  const columns: TableColumn[] = [
-    { title: 'User', field: 'createdBy' },
-    { title: 'Message', field: 'message' },
-    { title: 'Created', field: 'createdAt' },
-    { title: 'Status', field: 'status' },
+  type RowData = {
+    createdBy: { name: string; avatar?: string };
+    message: string;
+    createdAt: string;
+    status: string;
+  };
+
+  const columns: TableColumn<RowData>[] = [
+    {
+      title: 'User',
+      customFilterAndSearch: (query, row) =>
+        row.createdBy.name?.includes(query),
+      render: row => {
+        return (
+          <Chip
+            avatar={
+              <Avatar alt={row.createdBy.name} src={row.createdBy.avatar} />
+            }
+            label={row.createdBy.name}
+            variant="outlined"
+            size="small"
+          />
+        );
+      },
+      customSort: (row1, row2) =>
+        row1.createdBy.name < row2.createdBy.name ? 1 : -1,
+      field: 'createdBy',
+    },
+    {
+      title: 'Message',
+      customFilterAndSearch: (query, row) => row.message?.includes(query),
+      render: row => {
+        return <OverflowTooltip text={row.message} line={2} placement="top" />;
+      },
+      field: 'message',
+    },
+    {
+      title: 'Created',
+      customFilterAndSearch: (query, row) => row.createdAt?.includes(query),
+      render: row => {
+        return row.createdAt;
+      },
+      field: 'createdAt',
+    },
+    {
+      title: 'Status',
+      customFilterAndSearch: (query, row) => row.status?.includes(query),
+      render: row => {
+        return (
+          <Chip
+            label={row.status}
+            style={{ backgroundColor: getColor(row.status) }}
+            size="small"
+            variant="default"
+          />
+        );
+      },
+      field: 'status',
+    },
     { title: 'Actions', field: 'actions' },
```

Comment:
I think `sorting` and `filtering` should be explicitly set to `false` for the `Actions` column - this changes the ui so the user does not think it is a sortable value.